site/media/css/main.css

```diff
@@ -250,10 +251,84 @@ ul#cats a:hover {
 #cats .location { background-image: url('/media/images/categories/location.png'); }
 #cats .transparency { background-image: url('/media/images/categories/transparency.png'); }
 
+#cats .inform { background-image: url('/media/images/categories/inform.png'); }
+#cats .notify { background-image: url('/media/images/categories/defaultb.png'),
+  linear-gradient(0deg, #EDF5FF, #EDF5FF); }
```

Comment:
What meaning is the user expected to draw from the different colors? Some seemed closer together than others, but I don't know if that's the intent or just how my eye happened to see it.